kotlin/internal/jvm/compile.bzl

```diff
@@ -274,8 +278,23 @@ def _fold_jars_action(ctx, rule_kind, toolchains, output_jar, input_jars, action
 
 def _resourcejar_args_action(ctx, extra_resources = {}):
     res_cmd = []
+
+    # Get the strip prefix from the File object if provided
+    strip_prefix = None
```

Comment:
https://github.com/bazelbuild/bazel/blob/fc2239dba7db6551439d4f1f96be8433d35fa2a1/src/main/java/com/google/devtools/build/lib/rules/java/JavaHelper.java#L97-L113

Bazel impl. for reference